### ISSUE TITLE
JavaScript Errror: TypeError: f.filter is not a function

### ISSUE THREAD

**Issue by Kilo59** [reallistic/pyloot#18]
Hello, this looks like a cool project.
I've been looking for a FastAPI compatible memory profiler for a while.

I'm not sure if this project is still maintained but I ran into this JavaScript error.
I see the UI appear for a fraction of a second and then it goes away.

![image](https://user-images.githubusercontent.com/13108583/116020265-f03c3880-a613-11eb-8ad4-991ca0c0dca1.png)

Tried to follow the examples in the readme.
https://github.com/Kilo59/coronavirus-tracker-api/blob/pyloot/app/profiling.py
https://github.com/Kilo59/coronavirus-tracker-api/blob/pyloot/app/main.py#L49-L50

I'm running it locally with `uvicorn`.
https://github.com/Kilo59/coronavirus-tracker-api/blob/pyloot/Pipfile#L44

**Comment by reallistic** [comment]
@Kilo59 Hello thanks for checking out the project!
I've downloaded your project and found that the issue is from having gzip encoding enabled explicitly. 
pyloot also does its own gzip encoding. Thus encoding was happening twice and malformed data was returned.
I've added an option to pyloot to disable gzip encoding. I've also made the error handling a bit more graceful so that a nice error message is displayed instead of a white screen.
README is updated with instructions. The new version should publish automatically soon. Give it a try and add a comment if it does not solve your issue.
